common/sgx/verifier.c

```diff
@@ -800,13 +846,26 @@ oe_result_t oe_sgx_verify_evidence(
             &sgx_endorsements));
 
         // Verify the quote now.
-        OE_CHECK(oe_verify_quote_with_sgx_endorsements(
-            report_body,
-            report_body_size,
-            &sgx_endorsements,
-            time,
-            &valid_from,
-            &valid_until));
+        OE_CHECK_NO_TCB_LEVEL(
+            result_verify_quote,
+            oe_verify_quote_with_sgx_endorsements(
+                report_body,
+                report_body_size,
+                &sgx_endorsements,
+                time,
+                &platform_tcb_level,
+                &valid_from,
+                &valid_until));
+
+        if ((result_verify_quote == OE_OK) !=
```

Comment:
Suggest adding comment here to explain why `result_verify_quote != OE_OK)` and `platform_tcb_level.status.fields.up_to_date != 1` is OK.